sippy-ng/src/component_readiness/ComponentReadiness.js

```diff
@@ -199,7 +199,7 @@ export default function ComponentReadiness(props) {
   const now = new Date()
 
   // Sample is last 7 days by default
-  const initialSampleStartTime = new Date(now.getTime() - 7 * days)
+  const initialSampleStartTime = new Date(now.getTime() - 6 * days)
```

Comment:
I think you need to have similar fixes for the GA date releases, that's coming up as 29 days still.

baseEndTime=2023-10-31T23:59:59Z
baseStartTime=2023-10-03T00:00:00Z. <-- this should be the 4th

You'll probably need the change here and in ReleaseSelector.js

Reply:
thanks for the tip and the fix is pushed.

Reply:
I don't see the fix, it's below these lines on line 205 and 206.